designer/server/src/main/scala/pl/touk/nussknacker/ui/api/AppResources.scala

```diff
@@ -72,13 +73,13 @@ class AppResources(config: Config,
       path("healthCheck" / "process" / "deployment") {
         get {
           complete {
-            notRunningProcessesThatShouldRun.map[Future[HttpResponse]] { set =>
+            processesWithProblemStateStatus.map[Future[HttpResponse]] { set =>
               if (set.isEmpty) {
                 createHealthCheckHttpResponse(OK)
               } else {
-                logger.warn(s"Scenarios not running: ${set.keys}")
-                logger.debug(s"Scenarios not running - more details: $set")
-                createHealthCheckHttpResponse(ERROR, Some("Deployed scenarios not running (probably failed)"), Some(set.keys.toSet))
+                logger.warn(s"Alerting scenarios: ${set.keys}")
+                logger.debug(s"Alerting scenarios: $set")
+                createHealthCheckHttpResponse(ERROR, Some("Alerting scenarios"), Some(set.keys.toSet))
```

Comment:
`Alerting scenarios` => can we change message to be more descriptive? "Scenarios in problem status" / "..in invalid state"

Reply:
Fixed.